src/content/pages/beginners-day.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Beginners' Day
+url: /beginners-day
+subtitle:
+  This year we're hosting a special day for Python beginners.
+---
+
+## EuroPython Beginners' Day
+
+On Saturday 19th July, we'll be hosting a day to help introduce people to Python and its applications which will feature three tracks running in parallel. The events are designed to welcome newcomers to the Python ecosystem, including a series of talks and panels by junior developers and two workshops designed to introduce complete beginners to web development and data science.
+
+EuroPython Beginners' Day is open to everyone, from high school students or late career changers. Entry is free for EuroPython ticket holders, and €5 otherwise. Tickets for the Beginners' Day can be purchased here.
+
+Please sign up for the Beginners' Day and the specific track you're interested in using [this form](https://docs.google.com/forms/d/e/1FAIpQLSddM9RqRYc9jglmF-6vORbnFQ-_ymONU2cugUgB9poqqzLp5w/viewform?usp=dialog). The number of places available for each track is as follows:
+* **Beginners' Day panels and workshops**: X places
+* **Django Girls**: X places
+* **Humble Data**: 18 places
+
+### Beginners' Day panels and workshops
```

Comment:
personally, I'd consider putting the word `unconference` somewhere, because to me the content isn't really that structured and is more informal, rather than formal conference-like talks

Reply:
I agree! 😁